### PR TITLE
Fix many typos

### PR DESCRIPTION
**Pull request by goggle**
Fixes many typos found in the code base.

- - -
This change is [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/4486)

### CODE REVIEW

File: gui/locales/messages.pot [hunk 1]
```diff
@@ -1103,12 +1103,12 @@ msgstr ""
 
 #. This error message is shown if the user tries to launch an app that
 #. doesn't exist.
-#. This error message is shown if an application failes during startup.
+#. This error message is shown if an application fais during startup.
```
Comment (MrChocolatine) [inline gui/locales/messages.pot:1106]:
```suggestion
#. This error message is shown if an application fails during startup.
```

**Review by dlon — APPROVED** [review]

**Comment by faern** [comment]
Thank you for fixing all these typos! We'll look at having it merged asap.

**Review by albin-mullvad — APPROVED** [review]
> Reviewed 55 of 56 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/4486)* status: all files reviewed, 1 unresolved discussion (waiting on @goggle)

File: docs/architecture.md [hunk 2]
```diff
@@ -241,7 +241,7 @@ On Linux, connectivity is inferred by checking if there exists a route to a publ
 Currently the Mullvad API IP is used, but the actual IP does not matter as long as it's not a local
 one. This is done via Netlink and the route is queried via the exclusion firewall mark - otherwise,
 when a tunnel is connected, the address would always be routable as it'd be routed through the
-tunnel interface. As such, the offline monitor is somewhat coupled to routing and split tunelling on
+tunnel interface. As such, the offline monitor is somewhat coupled to routing and split tunnelling on
```
Comment (raksooo) [inline docs/architecture.md:244]: This line now is above the 100 character limit. Could you move the "on" down to the next line?

**Review by raksooo — COMMENTED** [review]
Wow that's a lot of typos! Thanks for fixing them!

**Review by raksooo — APPROVED** [review]

**Pull request merged**